tests/new_functional/spec/numbers.spec.js

```diff
@@ -57,4 +57,15 @@ describe('NumericRange', function() {
       });
   });
 
+  it('Given a range is set when the maximum is less than the minimum then a run time error should occur', function() {
+    return helpers.openQuestionnaire(number_schema)
+      .then(() => {
+        return browser
+          .setValue(SetMinMax.setMinimum(), '10')
+          .setValue(SetMinMax.setMaximum(), '0')
+          .click(SetMinMax.submit())
+          .getText('#main > h1').should.eventually.contain('Error 500');
```

Comment:
Do we want to be testing this sort of things as a functional test?
Seems to me this is more the sort of thing that can be tested with a unit or integration test.

Reply:
@jonnyshaw89 Removed the functional test as we already have a unit test test_number_range_validator